src/patternfly/components/Page/page.scss

```diff
@@ -155,22 +171,39 @@
   --pf-c-page__main-section--m-dark-100--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
   --pf-c-page__main-section--m-dark-200--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
 
+  // Drawer
+  --pf-c-page__drawer--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-page__drawer--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-page__drawer--BoxShadow: var(--pf-global--BoxShadow--md-left);
+  --pf-c-page__drawer--GridArea: main;
+  --pf-c-page__drawer--md--GridArea: drawer;
+  --pf-c-page__drawer--Width: 100%;
+  --pf-c-page__drawer--md--Width: #{pf-size-prem(450px)};
```

Comment:
@maryshak1996 I remember us talking about the width of the drawer being the same as the width of the sidebar, how do you feel about this width?